torch/_export/converter.py

```diff
@@ -343,17 +362,27 @@ def convert_graph_outputs(self):
             output_name = graph_output.debugName()
             if output_name in self.name_to_node:
                 args.append(self.name_to_node[output_name])
+                self.output_specs.append(
+                    OutputSpec(
+                        OutputKind.USER_OUTPUT,
+                        arg=TensorArgument(name=output_name),
+                        target=output_name,
+                    )
+                )
+            elif output_name in self.constant_map:
+                args.append(self.constant_map[output_name])
+                self.output_specs.append(
+                    OutputSpec(
```

Comment:
Does PT2 IR allow graph returning constant?

Reply:
I don't think we do right now 😅